doc/MONSTER_SPECIAL_ATTACKS.md

```diff
@@ -1,50 +1,69 @@
-- [Special Attacks](#special-attacks)
+# How to add special attacks to monsters
+- [Monster special Attacks](#monster-special-attacks)
     - [TODO](#todo)
-    - [Adding "special_attacks" to monsters](#adding-special_attacks-to-monsters)
-    - [Hardcoded Special Attacks](#hardcoded-special-attacks)
-    - [JSON Special Attacks](#json-special-attacks)
-        - ["monster_attack"](#monster_attack)
-        - ["bite"](#bite)
-        - ["leap"](#leap)
-        - ["gun"](#gun)
-    - [Monster Defense Attacks](#monster-defense-attacks)        
-
-# Special Attacks
-Some special attacks are also valid use actions for tools and weapons.
-See `monsters.json` for examples on how to use these attacks.
-Also see `monster_attacks.json` for more special attacks, for example, impale and scratch.
+    - [Adding `special_attacks` to monsters](#adding-special_attacks-to-monsters)
+      - [Old style array](#old-style-array)
+      - [New style object](#new-style-object)
+      - [Combination](#combination)
+    - [Hardcoded special attacks](#hardcoded-special-attacks)
+    - [JSON special attacks](#json-special-attacks)
+    - [Partially hardcoded special attacks](#partially-hardcoded-special-attacks)
+        - [`bite`](#bite)
+        - [`gun`](#gun)
+        - [`leap`](#leap)
+    - [Monster defensive attacks](#monster-defensive-attacks)        
+
+# Monster special attacks
+
+Monster creatures in C:DDA not only can do simple melee attacks, they have a wide array of attack-type actions.  For example, the common `GRAB` is a monster special attack.
+
+Depending on the intended effect, these can be valid `use_actions` for tools and weapons, hardcoded special attacks, any normal physical attack or even spells.  Also, depending on the kind of attack, these can be cooldown-based, conditioned or occur on death.
+
 
 ## TODO
 
-- Descriptions for `Hardcoded special attacks` could stand to be more descriptive of exactly what the attack does.
+- Descriptions for [Hardcoded special attacks](#hardcoded-special-attacks) could stand to be more descriptive of exactly what the attack does.
+
 
-## Adding "special_attacks" to monsters
-(array of special attack definitions, optional)
+## Adding `special_attacks` to monsters
 
-Monster's special attacks. This should be an array, each element of it should be an object (new style) or an array (old style).
+Monster special attacks can be defined either as old style arrays, new style objects, or a combination of both.
```

Comment:
Might want to add note that using old style should be avoided, I guess we probably want to phase it out eventually.

Reply:
Adding a suggestion in the doc.